recipe/meta.yaml

```diff
@@ -25,11 +24,12 @@ requirements:
 
 test:
   commands:
-    - test -f "$PREFIX/include/uv.h"
+    - test -f "$PREFIX/include/uv.h"    # [unix]
     - test -f "$PREFIX/lib/libuv.a"     # [unix]
     - test -f "$PREFIX/lib/libuv.so.1"  # [linux]
     - test -f "$PREFIX/lib/libuv.1.dylib"  # [osx]
     - test -f "$PREFIX/lib/libuv${SHLIB_EXT}"  # [not win]
+    - if not exist %LIBRARY_PREFIX%\include\uv.h (exit 1)  # [win]
```

Comment:
```suggestion
    - if not exist %LIBRARY_INC%\uv.h exit 1      # [win]
    - if not exist %LIBRARY_LIB%\uv.lib exit 1    # [win]
    - if not exist %LIBRARY_LIB%\uv.dll exit 1    # [win]
    - if not exist %LIBRARY_LIB%\uv_a.lib exit 1  # [win]
```

Reply:
Acc. to the build logs, the libraries are installed into `Library/lib/Debug`, so it looks like some CMake flags need to be tuned.